sonic-slave-stretch/Dockerfile.j2

```diff
@@ -298,7 +298,9 @@ RUN apt-get update && apt-get install -y \
         libprotobuf-dev \
 # For DHCP Monitor tool
         libexplain-dev \
-        libevent-dev
+        libevent-dev \
+# For SWI Tools
+	python-m2crypto
```

Comment:
>python-m2crypto [](start = 1, length = 15)

You can install python dependency by setup.py or pip. We prefer pypi packages than deb packages.

Reply:
If python-m2crypto not installed, cannot install M2crypto.

Reply:
I just tried, I have no python-m2crypto deb package, I can `pip install M2crypto`

Reply:
Did you try it on buster and stretch?

Reply:
If the python-m2crypto, then got following error when run "install pip M2Crypto"
 compilation terminated.
  error: command 'x86_64-linux-gnu-gcc' failed with exit status 1
  
  ----------------------------------------
  Failed building wheel for M2Crypto